src/mercator/mercator/static/js/Packages/Mercator/2015/Proposal/Module.ts

```diff
@@ -33,19 +34,24 @@ export var register = (angular) => {
             AdhImageModule.moduleName,
             AdhInjectModule.moduleName,
             AdhLocaleModule.moduleName,
+            AdhMetaApiModule.moduleName,
             AdhPermissionsModule.moduleName,
             AdhPreliminaryNamesModule.moduleName,
             AdhResourceAreaModule.moduleName,
             AdhResourceWidgetsModule.moduleName,
             AdhStickyModule.moduleName,
             AdhTopLevelStateModule.moduleName
         ])
-        .config(["adhResourceAreaProvider", "adhConfigProvider", (adhResourceAreaProvider, adhConfigProvider) => {
+        .config(["adhResourceAreaProvider", "adhConfigProvider", "adhMetaApiProvider", (
```

Comment:
The name `adhMetaApiProvider` is automatically generated by angular (`serviceName + 'Provider'`). We can not inject any services here because we are in configuration phase.